AppImage-package/bundle.sh

```diff
@@ -1,86 +1,115 @@
 #!/bin/bash
+
 ########################################################################
 # Package the binaries built on Travis-CI as an AppImage
 # By Simon Peter 2016
 # For more information, see http://appimage.org/
 ########################################################################
+
 sudo apt-get -y install coreutils binutils python-tk
+
 export ARCH=$(arch)
+
 APP=Scribus
-LOWERAPP=${APP,,}
+LOWERAPP=${APP,,} 
```

Comment:
Can you please remove the trailing white-spaces introduced in this commit (like the one in this line). They are unnecessary and `git diff` etc. will mark them with an alarming red colour.